.gitignore

```diff
@@ -7,7 +7,7 @@ build/
 .doctrees/
 .vscode/
 piccolo.egg-info/
-build/
```

Comment:
Looks like this was removed by mistake.

Reply:
I intentionally removed it. There is another one on line 6